cli/endpoints/amlarc/managed/canary-declarative-flow/2-scale-blue.yaml

```diff
@@ -0,0 +1,27 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/latest/onlineEndpoint.schema.json
+auth_mode: aml_token
+deployments:
+- code_configuration:
+    code:
+      local_path: ../../model-1/onlinescoring/
+    scoring_script: score.py
+  environment:
+    conda_file: file:../../model-1/environment/conda.yml
+    docker:
+      image: mcr.microsoft.com/azureml/intelmpi2018.3-ubuntu16.04:20210301.v1
+    path: .
+  model:
+    local_path: ../../model-1/model/sklearn_regression_model.pkl
+    name: model-1
+    version: 1
+  name: blue
+  scale_settings:
+    min_instances: 2
+    scale_type: auto
+identity:
+  type: system_assigned
+name: my-endpoint
+target: azureml:/subscriptions/589c7ae9-223e-45e3-a191-98433e0821a9/resourceGroups/localtest-r6-amlarc-rg/providers/Microsoft.MachineLearningServices/workspaces/localtest-r6-amlarc-ws/computes/localtest-r6-aml
+traffic:
```

Comment:
How about keep the same order of all settings?

Reply:
Yamls have been reordered